releasing/README.md

```diff
@@ -358,6 +358,23 @@ refreshMaster
 testKustomizeRepo
 ```
 
+### Publish Official Docker Image
+
+[k8s.io]: https://github.com/kubernetes/k8s.io
+[k8s-staging-kustomize]: https://pantheon.corp.google.com/home/dashboard?project=k8s-staging-kustomize
+
+Fork and clone the [k8s.io] repo.
+
+Checkout a new branch.
+
+Edit file `k8s.gcr.io/images/k8s-staging-kustomize/images.yaml`
+to add the new kustomize version and the image sha256.
+
+Image sha256 can be found in the GCP project [k8s-staging-kustomize].
```

Comment:
Can you give more details where to find the image sha256? I went to the gcp project linked here and am unsure where to look for the sha256